src/Microsoft.TestPlatform.Utilities/ClientUtilities.cs

```diff
@@ -43,6 +44,12 @@ public static void FixRelativePathsInRunSettings(XmlDocument xmlDocument, string
         {
             FixNodeFilePath(resultsDirectoryNode, root);
         }
+
+        var dotnetHostPathNode = xmlDocument.SelectSingleNode(DotnetHostPathXPath);
```

Comment:
Tests are failing because SelectSingleNode is likely case sensitive?

Reply:
No test is failing on CI. Is there any change you would have forgot to push?

Reply:
Check previous CI runs, the problem was with the casing. If I used `<RunConfiguration><DotnetHostPath>` instead of `DotNetHostPath` it would break.

Reply:
Thanks! We are planning on improving the way we read/write values in runsettings so I guess this could be handled here.